scripts/install/get_tendermint.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+#set -x
+
+# an examplary download-url
+# https://github.com/tendermint/tendermint/releases/download/v0.34.13/tendermint_0.34.13_linux_amd64.tar.gz
+export TM_MAJORMINOR="0.34"
+export TM_PATCH="13"
+export TM_REPO="https://github.com/tendermint/tendermint"
+
+export TM_VERSION="${TM_MAJORMINOR}.${TM_PATCH}"
+
+export TARGET_PATH="/usr/local/bin"
+export TMP_PATH="/tmp"
+
+error_exit()
+{
+    echo "Error: $1"
+    exit 1
+}
+
+# check for existence
+export TM_EXECUTABLE=$(which tendermint)
+if [ -x ${TM_EXECUTABLE} ]; then
```

Comment:
In bash 5.0.17, this was still printing out the error. This seems to work:
```suggestion
if [ -x "$TM_EXECUTABLE" ]; then
```

Reply:
hello tomas,
 
yes, you are right - just push the change to the feature-branch or should i do it?
 
br,
joachim

Reply:
I'll commit it from here, thanks!